Tests/CommandsTests/TestCommandTests.swift

```diff
@@ -278,7 +266,7 @@ struct TestCommandTests {
                 )
             }
         } when: {
-            buildSystem == .swiftbuild && ProcessInfo.hostOperatingSystem == .windows
+            buildSystem == .swiftbuild
```

Comment:
issue (possibly-blocking): this seems like a regression as this is now failing on all platforms, while it was previously only failing on Windows.  Why is the plan to address this?